contracts/FeralfileArtworkV3.sol

```diff
@@ -425,10 +453,36 @@ contract FeralfileExhibitionV3 is ERC721Enumerable, Authorizable, IERC2981 {
         emit NewArtworkEdition(owner_, artworkID_, editionID);
     }
 
+    /// @notice burn editions
+    /// @param editionIDs_ - the list of edition id will be burned
+    function burnEditions(uint256[] memory editionIDs_) public {
+        require(isBurnable, "FeralfileExhibitionV3: not allow burn edition");
+
+        for (uint256 i = 0; i < editionIDs_.length; i++) {
+            require(
+                _exists(editionIDs_[i]),
+                "ERC721: artwork edition is not found"
+            );
+            require(
+                _isApprovedOrOwner(_msgSender(), editionIDs_[i]),
+                "ERC721: caller is not token owner nor approved"
+            );
+            ArtworkEdition storage edition = artworkEditions[editionIDs_[i]];
```

Comment:
I think 
```
ArtworkEdition memory edition
```
might save a little bit of cost

Reply:
Fixed